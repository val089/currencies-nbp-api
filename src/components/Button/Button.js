import React from 'react';
import './Button.scss';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';

const Button = ({ onClick, icon }) => {
	return (
		<button className="button" onClick={onClick}>
			<FontAwesomeIcon icon={icon} />
		</button>
	);
};

export default Button;
