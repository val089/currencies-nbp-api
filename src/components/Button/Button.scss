@import 'colors.scss';

.button {
    font-size: 2rem;
    color: $add-to-favorites-btn;
    outline: none;
    transition: .4s;

    &:hover {
       color: red;
       transform: scale(1.3);
    }
}