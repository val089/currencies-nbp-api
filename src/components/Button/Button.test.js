import React from 'react';
import renderer from 'react-test-renderer';
import Button from './Button';

it('renders ok', () => {
	const tree = renderer.create(<Button className="button" />).toJSON();
	expect(tree).toMatchSnapshot();
});
