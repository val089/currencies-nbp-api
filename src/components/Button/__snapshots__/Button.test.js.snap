// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ok 1`] = `
<button
  className="button"
/>
`;
